PROCESS.md

```diff
@@ -7,7 +7,7 @@ this document outlines the process that should be followed in proposing, plannin
 Features and ideas are thrown around in our various discussions all the time. The purpose of defining this process is to define a concrete progression into which we can funnel these ideas so that we can have discussions and make real progress to their implementation and completion.
 
 ## Process
-1. A feature, project, curriculum addition, etc. is discussed and selected for development.
+1. A feature, project, curriculum addition, etc. is discussed and selected for development and is created as an issue on "discussions" for further expansion.
```

Comment:
I think this should be an entirely new number.

```
1. A feature ...
2. An issue is created in the "discussions" repository for further discussion and details.
3. A person or small ...
```

Reply:
agree with @mindovermiles262

Reply:
I've briefly thought about that, Fixed now.